core/src/components/input/test/bottom-content/input.e2e.ts

```diff
@@ -105,12 +105,13 @@ configs({ modes: ['md', 'ionic-md'], directions: ['ltr'] }).forEach(({ title, sc
         await expect(errorText).toBeVisible();
         await expect(errorText).toHaveText('my error');
       });
-      test('error text should change when variable is customized', async ({ page }) => {
+      test('error text and highlight should change when variable is customized', async ({ page }) => {
         await page.setContent(
           `
           <style>
             ion-input.custom-input {
               --highlight-color-invalid: purple;
+              --text-color-invalid: purple; // ionic only
```

Comment:
Should we be seeing a screenshot with purple text somewhere? 👀

Reply:
I was wondering this too 👀

Reply:
Added: https://github.com/ionic-team/ionic-framework/pull/29291/commits/7baf62ab0f1f89491538238abaefe8b0058cd6de and https://github.com/ionic-team/ionic-framework/pull/29291/commits/ecfc9aaf598806f344c637cab6a50abc24c63aa5